diff --git a/prepare_vscode.sh b/prepare_vscode.sh
index 9bda24c..4b5f14e 100755
--- a/prepare_vscode.sh
+++ b/prepare_vscode.sh
@@ -27,7 +27,7 @@ releaseNotesUrl='setpath(["releaseNotesUrl"]; "https://go.microsoft.com/fwlink/?
 keyboardShortcutsUrlMac='setpath(["keyboardShortcutsUrlMac"]; "https://go.microsoft.com/fwlink/?linkid=832143")'
 keyboardShortcutsUrlLinux='setpath(["keyboardShortcutsUrlLinux"]; "https://go.microsoft.com/fwlink/?linkid=832144")'
 keyboardShortcutsUrlWin='setpath(["keyboardShortcutsUrlWin"]; "https://go.microsoft.com/fwlink/?linkid=832145")'
-quality='setpath(["quality"]; "stable")'
+quality='setpath(["quality"]; "insider")'
 extensionsGallery='setpath(["extensionsGallery"]; {"serviceUrl": "https://open-vsx.org/vscode/gallery", "itemUrl": "https://open-vsx.org/vscode/item"})'
 linkProtectionTrustedDomains='setpath(["linkProtectionTrustedDomains"]; ["https://open-vsx.org"])'
 nameShort='setpath(["nameShort"]; "VSCodium")'
